### PR TITLE
fix(argo-cd): update initial password guidance in NOTES

### PR DESCRIPTION
**Pull request by valerauko**
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [ ] Any new values are backwards compatible and/or have sensible default.
* [ ] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [x] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

fixes https://github.com/argoproj/argo-helm/issues/706

I noticed that the guidance in NOTES regarding the initial admin password isn't true anymore for versions 1.9 on.

### CODE REVIEW

File: charts/argo-cd/templates/NOTES.txt [hunk 1]
```diff
@@ -9,7 +9,8 @@ In order to access the server UI you have the following options:
       - Add the `--insecure` flag to `server.extraArgs` in the values file and terminate SSL at your ingress: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/ingress.md#option-2-multiple-ingress-objects-and-hosts
 
 
-After reaching the UI the first time you can login with username: admin and the password will be the
-name of the server pod. You can get the pod name by running:
+After reaching the UI the first time you can login with username: admin and the random password generated during the installation. You can find the password by running
```
Comment (valerauko) [inline charts/argo-cd/templates/NOTES.txt:12]: Should there be a version check (global.image.tag maybe?) to match Argo CD's version or is this (a sensible default) enough?

Comment (mkilchhofer) [inline charts/argo-cd/templates/NOTES.txt:12]:
```suggestion
After reaching the UI the first time you can login with username: admin and the random password generated during the installation. You can find the password by running:
```

Comment (mkilchhofer) [reply charts/argo-cd/templates/NOTES.txt]: I think we can proceed with the new text only. The chart probably only works with 2.x now. At least for Dex there are command arguments which doesn't work for pre 2.x

Comment (mkilchhofer) [reply charts/argo-cd/templates/NOTES.txt]: Grml, we now need to reword the commit message :-( The DCO bot complains about missing `Signed-off-by: Vale <valerauko@gmail.com>`. Would you mind to reword this?

Comment (mkilchhofer) [reply charts/argo-cd/templates/NOTES.txt]: I fixed it myself :-)

**Review by mkilchhofer — APPROVED** [review]

**Pull request merged**